Bugsnag/BugsnagSystemState.m

```diff
@@ -18,58 +18,39 @@
 
 #import "BSGFileLocations.h"
 #import "BSGJSONSerialization.h"
+#import "BSGRunContext.h"
 #import "BSGUtils.h"
 #import "BSG_KSCrashState.h"
-#import "BSG_KSMach.h"
 #import "BSG_KSSystemInfo.h"
 #import "BSG_RFC3339DateTool.h"
-#import "BugsnagKVStoreObjC.h"
 #import "BugsnagLogger.h"
 #import "BugsnagSession+Private.h"
-#import "BugsnagSessionTracker.h"
-#import "BugsnagSystemState.h"
 
 #import <stdatomic.h>
 
+#define SYSTEMSTATE_APP_VERSION @"version"
+#define SYSTEMSTATE_APP_BUNDLE_VERSION @"bundleVersion"
+
+#define SYSTEMSTATE_DEVICE_BOOT_TIME @"bootTime"
+
 static NSString * const ConsecutiveLaunchCrashesKey = @"consecutiveLaunchCrashes";
```

Comment:
Not for this PR, but our styling and define vs const should be harmonized.

Reply:
Agreed - this will be removed in the next PR btw